src/PHPStan/Collector/TemplateContextCollector.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace TwigStan\PHPStan\Collector;
+
+use PhpParser\Node;
+use PHPStan\Collectors\Collector;
+
+/**
+ * @phpstan-type TemplateData = array{
+ *      startLine: int,
+ *      endLine: int,
+ *      template: string,
+ *      context: string,
+ *  }
+ *
+ * @template-covariant TNodeType of Node
+ * @extends Collector<TNodeType, non-empty-array<TemplateData>>
```

Comment:
```suggestion
 * @extends Collector<TNodeType, non-empty-list<TemplateData>>
```
See https://github.com/twigstan/twigstan/pull/21